src/app/events/components/tests/tests-list/tests-list.component.html

```diff
@@ -29,7 +29,8 @@
         [ngClass]="{
           'visible-on-mobile': !hasTests || !selectedTest,
         }"
-        href="{{ buildLinkToRatingOverview() | async }}"
+        [routerLink]="['/events', course?.Id, 'evaluation']"
+        [queryParams]="{ returnlink: '/events/' + course?.Id + '/tests' }"
```

Comment:
Die `buildLinkToRatingOverview` Methode solltest du noch löschen, das ist mit dieser Änderung dead Code...

Und dann würde ich noch `course` in die `@if` Condition auf Zeile 26 aufnehmen, weil es macht ja meiner Meinung nach keinen Sinn, den Link darzustellen, wenn `course === null`. Dann kannst du auch die `?.` sein lassen.